content/applications/productivity/discuss/advanced/email_servers.rst

```diff
@@ -224,6 +224,24 @@ alias in your mail server.
 
 .. _Office 365 documentation:
     https://support.office.com/en-us/article/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-office-365-69f58e99-c550-4274-ad18-c805d654b4c4
+    
+Set up different outgoing servers in a multi-company environment
+================================================================
+
+The "From Filter" allows to use a specific outgoing depending of the "From". If an email is sent from Odoo, the email server will be chosen as follows:
+
+#.  Odoo searches for an email server having the same "From Filter" as the "From" defined in the outgoing email. For example, if the "From" is foo@bar.com, only the email servers having the "From Filter" equals to foo@bar.com will be returned.
+
+#. If not found, Odoo searches for an email server having the same domain in the "From Filter" as in the "From". For example, if the "From" is foo@bar.com, only the email servers having the "From Filter" equals to bar.com will be returned.
+
+If no email servers are found after checking for the domain, then Odoo will return all email servers without any "From Filter" set.
+
+Should this previous query return no results, then Odoo performs a search for an email server using the system parameter: 'mail.default.from'. First, the email address listed will attempt to match an email server and then the domain will attempt a match.
+
+If no email server is found then Odoo will return the first outgoing email server (sorted by priority). In this case, it is likely the email configuration will be spoofing the email and it will be flagged as Spam. 
+
+.. note:: If several email servers are found, Odoo will use the first one per priority.
```

Comment:
Please put the content of note/warning/etc. directives below the directive:
```rst
.. note::
   If several email servers...
```
Otherwise it is interpreted as the title of the directive and is harder to maintain.